components/editor/route/InfoEdit.vue

```diff
@@ -565,11 +565,12 @@ export default {
               //grade modifier doesn't match
               check = false;
             }
+
             if (
-              (this.routeList[this.panel].pitches[pi].danger &&
+              (this.routeState[this.panel].pitches[pi].danger &&
                 this.routeList[this.panel].pitches[pi].danger !==
                   this.routeState[this.panel].pitches[pi].danger) ||
-              (!this.routeList[this.panel].pitches[pi].danger &&
+              (!this.routeState[this.panel].pitches[pi].danger &&
```

Comment:
these guys were just typo-ed, it looks like; matches the logic for `gradeModifier` this way, which seems to make sense